src/app/components/typography.tsx

```diff
@@ -97,19 +97,29 @@ const titleStyles = (as: Headings) => {
   }
 };
 
-export const Title = forwardRefWithAs<BoxProps, Headings>((props, ref) => (
-  <BaseText
-    userSelect="none"
-    letterSpacing="-0.01em"
-    fontFamily="'Open Sauce One'"
-    fontWeight="500"
-    color={color('text-title')}
-    ref={ref}
-    display="block"
-    css={titleStyles(props.as)}
-    {...props}
-  />
-));
+export const Title = forwardRefWithAs<BoxProps, Headings>((props, ref) => {
+  // These next two lines are to prevent tall graphemes (like emojis, some
+  // chinese characters, and others) getting cut off by pseudoelements inserted
+  // by capsize, which is due for removal:
+  // https://github.com/hirosystems/stacks-wallet-web/issues/2344
+  const styleObject = titleStyles(props.as);
+  const withoutBeforePseudoElement = { ...styleObject, ['::before']: undefined };
```

Comment:
This is to prevent emojis and tall chars being cut off. Admittedly a bit of a hack, although seems acceptable given we're going to [remove capsize](https://github.com/hirosystems/stacks-wallet-web/issues/2344)

![image](https://user-images.githubusercontent.com/116000646/197607825-a41d4eb6-330a-4720-b69e-5633574a1b49.png)

Reply:
Scrapped this approach, used prop instead